composer.json

```diff
@@ -49,6 +50,7 @@
         ]
     },
     "autoload-dev": {
+        "psr-0": { "Twig_": "vendor/twig/twig/lib/" },
```

Comment:
Twig 1.3 has no autoload... should I drop support for it? This is ugly...

Reply:
Wait I can't, because the dependency is not required, so Composer cannot protect people who want to upgrade... next major.

Reply:
What about using the `conflict` section for this?

Reply:
Btw. Twig 1.3 was released looooong time ago https://github.com/twigphp/Twig/releases?after=v1.6.2

Reply:
The dependency is optional, so that would mean forcing people to upgrade twig even if they do not use that feature we propose... but indeed, that's a long time so maybe just drop it indeed?
Another thing to consider: we might release cachebundle 3 today. If we do, I will drop 1.3 (and 1.x) on master, as well as rename `TwigTemplate14` to just `TwigTemplate`, effectively grouping the BC-breaks together. That means the result will be the same, except that the last 2.x release will not break anything.

Reply:
> The dependency is optional, so that would mean forcing people to upgrade twig even if they do not use that feature we propose... 

That's a general problem with optional deps. We can guarantee if the user is using an optional feature and needs a specific versions or not. I would risk adding a conflict section and live with a higher version instead of having a dependency version problem.

Reply:
Even better idea: let's do all of this in `master` instead directly. No need to risk anything on the last version of the stable branch. People that want to use twig 2 can already do so: the `TwigTemplate14` class is compatible.

Reply:
Guess what? You now have a wonderful tool to check package version since we support PHP 7 only: https://packagist.org/packages/ocramius/package-versions

With love.

Reply:
Guess what? You now have a wonderful tool to check package version since we support PHP 7 only: https://packagist.org/packages/ocramius/package-versions

With love.